components/mastodon/sources/new-status/new-status.mjs

```diff
@@ -0,0 +1,52 @@
+import common from "../common/base.mjs";
+
+export default {
+  ...common,
+  key: "mastodon-new-status",
+  name: "New Status",
+  description: "Emit new event when a new status is posted to your Profile",
```

Comment:
you could also add link to the docs of the currently used endpoint on this source